### PR TITLE
Add component tests

### PR DESCRIPTION
**Pull request by captainsafia**
Adding tests.

### CODE REVIEW

**Comment by captainsafia** [comment]
Gonna try and include tests for our epics in this as well. Reading up on it now.

File: test/renderer/components/cell/cell-spec.js [hunk 1]
```diff
@@ -35,4 +39,35 @@ describe('Cell', () => {
...
+      id: undefined,
+    });
+
+    inst.focusBelowCell();
+    expect(store.dispatch.thirdCall).to.have.been.calledWith({
+      type: 'FOCUS_NEXT_CELL',
+      id: undefined,
+      createCellIfUndefined: true,
+    });
+  });
```
Comment (rgbkrk) [inline test/renderer/components/cell/cell-spec.js:72]: :cool:

**Comment by rgbkrk** [comment]
> Gonna try and include tests for our epics in this as well. Reading up on it now.

How about just keeping this as new focus cell tests and making a new PR?

**Comment by captainsafia** [comment]
Sure! Merge away!

**Pull request merged**